service_objects/fields.py

```diff
@@ -74,8 +88,10 @@ class ModelField(forms.Field):
     error_model_class = _("%(cls_name)s(%(model_class)r) is invalid.  First "
                           "parameter of ModelField must be either a model or a "
                           "model name.")
+    error_non_list = _("Object is not iterable.")
```

Comment:
This error isn't needed in ModelField since ModelField only takes one object, not a list

Reply:
Will do!